dissect/target/plugins/apps/texteditor/texteditor.py

```diff
@@ -0,0 +1,17 @@
+from dissect.target.helpers.descriptor_extensions import UserRecordDescriptorExtension
+from dissect.target.helpers.record import create_extended_descriptor
+from dissect.target.plugin import NamespacePlugin
+
+GENERIC_TAB_CONTENTS_RECORD_FIELDS = [
+    ("string", "content"),
+    ("string", "content_length"),
```

Comment:
No, not really. I'll leave it out. I initially figured it would be usefil in e.g. `target-query ... | rdump -s "r.content_length >= 200"` to look for long texts, but you can just do `"len(r.content) >= 2--"` as well..